### PR TITLE
WCOR-0020148: Update README with QA instructions.

### PR DESCRIPTION
**Pull request by lgbridges**
Added QA Workflow section. Replaced catchup commands with 'blt custom:catchup'. Various minor edits.

### CODE REVIEW

**Comment by gatoona** [comment]
Can we start incorporating anchor tags as our readme gets longer?
https://github.com/fefong/markdown_readme

**Comment by lgbridges** [comment]
> 
> 
> Can we start incorporating anchor tags as our readme gets longer?
> https://github.com/fefong/markdown_readme

Yes, I will review and set this up.

File: README.md [hunk 1]
```diff
@@ -63,227 +78,295 @@ key (/home/ide/.ssh/id_ed25519):* -- just **press Enter** to accept the default.
...
+    
+    - Move the item to the *Work in progress* column of the sprint tracking board.
+    
+    If *all* conditions of the test plan are satisfied:
+    
+    - In the ***Review*** pane of the GitHub PR dashboard, add any comments you have, and @-notify both the developer and Build Coordinator. (Comments should include any quality improvement suggestions you have for the developer, regardless of passing the item.) 
+    
+        Select *Approve* below the review comment field and **Submit**.
+    
+    - Make a note in the ServiceIT story indicating that QA has passed.
```
Comment (gatoona) [inline README.md:362]: Can this be done in GitHub?

File: README.md [hunk 2]
```diff
@@ -63,227 +78,295 @@ key (/home/ide/.ssh/id_ed25519):* -- just **press Enter** to accept the default.
...
+    
+    - Leave yourself as the *Additional Assignee* in the ServiceIT story for future reference about who performed QA.
```
Comment (gatoona) [inline README.md:364]: Same as above. Can this be done in GitHub?

File: README.md [hunk 3]
```diff
@@ -63,227 +78,295 @@ key (/home/ide/.ssh/id_ed25519):* -- just **press Enter** to accept the default.
...
+5. Once you've accessed the CD environment site as admin, ensure the test plan can be replicated exactly. Note that the CD environment is separate from the main Dev site and all local IDEs. (If you break something in the site, the CD environment can be easily recreated -- contact a WSA if you need assistance.)
+
+6. If any condition of the test plan is ***not*** satisfied:
 
-4. Acquia Pipelines will run *again*---this time, on the primary Dev site build. As before, if no issues occur, the Pipelines test will pass. This will result in an automatic push of the code to the ***Acquia*** git repository for the Dev site (*pipelines-build-develop*) and code will automatically deploy to the primary Dev site.
+    - In the ***Review*** pane of the GitHub PR dashboard, make a comment regarding what part of the test plan failed, and @-notify the developer.
+    
+        Select *Request changes* below the review comment field and **Submit**.
+    
+    - Move the item to the *Work in progress* column of the sprint tracking board.
```
Comment (gatoona) [inline README.md:354]: This step seems redundant.

**Review by gatoona — CHANGES_REQUESTED** [review]

**Review by gatoona — APPROVED** [review]

**Pull request merged**